### PR TITLE
ENG-1519 limit bid

### PR DESCRIPTION
**Pull request by frank0170**

### CODE REVIEW

File: packages/features/sales-escrows/modals/StartEscrowModal.tsx [hunk 1]
```diff
@@ -151,8 +151,16 @@ export function StartEscrowModal({
           amount: `The minimum bid is ${minBid.toFixed()} ${odc.tokenSymbol}`,
         };
       }
+      if ((amount.isGreaterThan(toLargerUnit(odc.buyNow, token.decimals)))) {
```
Comment (Motokoder) [inline packages/features/sales-escrows/modals/StartEscrowModal.tsx:154]: Please change to else if

File: packages/features/sales-escrows/modals/StartEscrowModal.tsx [hunk 2]
```diff
@@ -151,8 +151,16 @@ export function StartEscrowModal({
...
+        errors = {
+          ...errors,
+          amount: `The maximum bid needs to be less than  ${toLargerUnit(odc.buyNow, token.decimals)} ${odc.tokenSymbol}`,
```
Comment (Motokoder) [inline packages/features/sales-escrows/modals/StartEscrowModal.tsx:157]:
```suggestion
          amount: `Bid must be less than buy now price (${toLargerUnit(odc.buyNow, token.decimals)} ${odc.tokenSymbol})`,
```

**Review by Motokoder — CHANGES_REQUESTED** [review]

File: packages/features/sales-escrows/modals/StartEscrowModal.tsx (hunk 1 above)
Comment (frank0170) [reply packages/features/sales-escrows/modals/StartEscrowModal.tsx]: done

File: packages/features/sales-escrows/modals/StartEscrowModal.tsx (hunk 2 above)
Comment (frank0170) [reply packages/features/sales-escrows/modals/StartEscrowModal.tsx]: done, thx

**Review by Motokoder — APPROVED** [review]

**Pull request merged**